index.html

```diff
@@ -36,6 +36,196 @@ <h1>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button type="button">See project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio3.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <h1>Multi-Post
+                        Stories</h1>
+                </div>
+                <ul class="projectTools">
+                    <li>CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button type="button">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
     
+    <section id="aboutme">
+        <div class="top">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="letconnect" href="">LET’S CONNECT</a> 
+            <ul class="socialIconsGrid">
+                <li><a href="#"><img src="./assets/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="./assets/linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="./assets/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="./assets/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
+            </ul>
+            <div class="action"><button type="button">Get my resume</button></div>
```

Comment:
- [ ] Good Job in implementing this task, however, As part of the project requirement, Please add the Button interactions in your styles like `enable, hover, etc`, you can check this [link](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) for reference on how to implement it. Below is the Figma design:

![btn Hover](https://github.com/BilalLiaquat7/Portfolio/assets/43528594/e033275d-65bd-4aed-a4c3-00c50bb79387)